api-specifications/retailmedia_2022-04.json

```diff
@@ -0,0 +1,6738 @@
+{
+    "openapi": "3.0.1",
+    "info": {
+        "title": "Criteo API",
+        "description": "Criteo publicly exposed API",
+        "version": "2022-04"
+    },
+    "servers": [
+        {
+            "url": "https://api.criteo.com"
+        }
+    ],
+    "paths": {
+        "/me": {
```

Comment:
Was the OAS generation fixed in order to generate this or was it added manually?

Reply:
Fixed, we were using the wrong endpoint :). We should use **/v1/open-api-specification/MarketingSolutions/2021-04** with body:
`{
    "overrides": {
        "exam_oauth": {
            "routes": [
                {
                    "gatewayPathTemplate": "/oauth2/token",
                    "httpMethod": "Post",
                    "apiVersion": "NoVersion"
                }
            ]
        },
        "exam_administration": {
            "routes": [
                {
                    "gatewayPathTemplate": "/me",
                    "httpMethod": "Get",
                    "apiVersion": "NoVersion"
                }
            ]
        },
        "audience_core": {},
        "exam_audience": {},
        "exam_campaign": {},
        "exam_config": {},
        "exam_creatives": {},
        "exam_mock_test": {},
        "exam_sample": {},
        "exam_sellers": {},
        "exam_statistics": {},
        "papi_mapi": {},
        "platform_advertiser": {},
        "pmc": {},
        "product_importer": {},
        "reco_configuration": {},
        "reco_filter_preview": {},
        "reco_onsite": {},
        "retail_media_accounts_api": {},
        "retail_media_audience_api": {},
        "retail_media_catalog_api": {},
        "retail_media_cerberus_api": {},
        "retail_media_data_export": {},
        "retail_media_miracle": {},
        "targeting_api": {},
        "twopac_targeting_api": {}
    }
}`